src/components/material/disclosures/disclosure.tsx

```diff
@@ -0,0 +1,35 @@
+import ExpandMoreIcon from "@reload/dpl-design-system/build/icons/collection/ExpandMore.svg";
+import React, { FC, ReactNode } from "react";
+import { useText } from "../../../core/utils/text";
+
+export interface DisclosureProps {
+  svgIcon: { path: string; altText: string };
+  title: string;
+  children?: ReactNode;
+}
+
+export const Disclosure: FC<DisclosureProps> = ({
+  title,
+  children,
+  svgIcon: { path, altText }
+}) => {
+  const t = useText();
+  return (
+    <details className="disclosure text-body-large">
+      <summary className="disclosure__headline text-body-large">
+        <div className="disclosure__icon bg-identity-tint-120 m-24">
+          <img className="disclosure__icon" src={path} alt={altText} />
```

Comment:
I'd definitely hide both img from screen readers -> alt=""

Reply:
But alt, in this line, comes from `svgIcon.altText`.
In the other image the alt text comes from t.
But on second thought I think both texts should come as parameters, since this component is a simple rendering component

Reply:
So. Now it is changed